RELEASE_NOTES.md

```diff
@@ -1,9 +1,109 @@
+
+Then when we perform our yearly major release we'll remove the preview designation from all resources, and the built in resources will take precedence over resources with the same names in cookbooks.
+
+## New Resources
+
+### chocolatey_config
+
+Use the chocolatey_config resource to add or remove Chocolatey configuration keys."
+
+#### Actions
+
+- `set` - Sets a Chocolatey config value.
+- `unset` - Unsets a Chocolatey config value.
+
+#### Properties
+
+- `config_key` - The name of the config. We'll use the resource's name if this isn't provided.
+- `value` - The value to set.
+
+### chocolatey_source
+
+Use the chocolatey_source resource to add or remove Chocolatey sources.
+
+#### Actions
+
+- `add` - Adds a Chocolatey source.
+- `remove` - Removes a Chocolatey source.
+
+#### Properties
+
+- `source_name` - The name of the source to add. We'll use the resource's name if this isn't provided.
+- `source` - The source URL.
+- `bypass_proxy` - Whether or not to bypass the system's proxy settings to access the source.
+- `priority` - The priority level of the source.
+
+### powershell_package_source
+
+Use the `powershell_package_source` resource to register a powershell package repository
+
+### Actions
+
+- `register` - Registers and updates the powershell package source.
+- `unregister` - Unregisters the powershell package source.
+
+#### Properties
+
+- `source_name` - The name of the package source
+- `url` - The url to the package source
+- `trusted` - Whether or not to trust packages from this source.
+- `provider_name` - The package management provider for the source. It supports the following providers: 'Programs', 'msi', 'NuGet', 'msu', 'PowerShellGet', 'psl' and 'chocolatey'.
+- `publish_location` - The url where modules will be published to for this source. Only valid if the provider is 'PowerShellGet'.
+- `script_source_location` - The url where scripts are located for this source. Only valid if the provider is 'PowerShellGet'.
+- `script_publish_location` - The location where scripts will be published to for this source. Only valid if the provider is 'PowerShellGet'.
+
+### kernel_module
+
+Use the kernel_module resource to manage kernel modules on Linux systems. This resource can load, unload, blacklist, install, and uninstall modules.
+
+#### Actions
+
+- `install` - Load kernel module, and ensure it loads on reboot.
+- `uninstall` - Unload a kernel module and remove module config, so it doesn't load on reboot.
+- `blacklist` - Blacklist a kernel module.
+- `load` - Load a kernel module.
+- `unload` - Unload kernel module
+
+#### Properties
+
+- `modname` - The name of the kernel module.
+- `load_dir` - The directory to load modules from.
+- `unload_dir` - The modprobe.d directory.
+
+### ssh_known_hosts_entry
+
+Use the ssh_known_hosts_entry resource to append an entry for the specified host in /etc/ssh/ssh_known_hosts or a user's known hosts file if specified.
```

Comment:
Append an entry -> Add an entry?  Append would imply an entry was appended on every run but I'm guessing this is idempotent.